src/api/projects.js

```diff
@@ -0,0 +1,45 @@
+import { GET, PUT, POST, DELETE } from "../utils/constants";
+import createApiRequest from "../utils/createApiRequest";
+
+export const readProjects = () => createApiRequest(GET, "/projects");
+
+export const createProject = (
+  name,
```

Comment:
Like on the backend, you will probably want to create a model (essentialy a DTO) for projects on the frontend. This way you could do something like:

```
export const createProject = (project) => createApiRequest(POST, '/projects', project);
```